packages/graphql/tests/integration/advanced-filtering.int.test.ts

```diff
@@ -680,6 +680,288 @@ describe("Advanced Filtering", () => {
                 })
             );
         });
+
+        test("should find Movies GT string", async () => {
+            const session = await neo4j.getSession();
+
+            const movieType = generateUniqueType("Movie");
+
+            const typeDefs = `
+                        type ${movieType.name} {
+                            title: String
+                        }
+                    `;
+
+            const neoSchema = new Neo4jGraphQL({
+                config: {
+                    features: {
+                        filters: {
+                            String: {
+                                LT: true,
+                                GT: true,
+                                LTE: true,
+                                GTE: true,
+                            },
+                        },
+                    },
+                },
+                typeDefs,
+            });
+
+            const matrix = "The Matrix";
+            const matrixReloaded = "The Matrix Reloaded";
+            const matrixRevolutions = "The Matrix Revolutions";
+
+            try {
+                await session.run(`
+                            CREATE (:${movieType.name} {title: $matrix})
+                            CREATE (:${movieType.name} {title: $matrixReloaded})
+                            CREATE (:${movieType.name} {title: $matrixRevolutions})
+                        `,
+                    { matrix, matrixReloaded,  matrixRevolutions }
+                );
+
+                const query = `
+                            {
+                                ${movieType.plural}(where: { title_GT: "${matrix}" }) {
```

Comment:
I may be missing another test case, but just to ensure this work as expected, it would be good to add a film with a smaller title (e.g "IT") so we can confirm that this filter is removing that one

Reply:
Oh, sorry, for some reason in my head I though it was length and not alphabetical order.

My point still is the same (only use something like aaa, or whatever) just to make sure that at least a field not greater than is filtered

Reply:
No problem!